README.md

```diff
@@ -9,100 +9,511 @@ support@appsflyer.com
 
 > *When submitting an issue please specify your AppsFlyer sign-up (account) email , your app ID , production steps, logs, code snippets and any additional relevant information.*
 
-### <a id="plugin-build-for"> This module is built for
+<!-- TOC start -->
+## Table Of Content
+  * [This Module Is Built For](#plugin-build-for)
+  * [Adding The SDK To Your Project](#install-sdk)
+  * [Basic Integration Of The SDK](#basic-integration)
+    + [Create PurchaseClient Instance ](#create-instance)
+    + [Start Observing Transactions](#start)
+    + [Stop Observing Transactions](#stop)
+    + [Log Subscriptions](#log-subscriptions)
+    + [Log In App](#log-inapps)
+  * [Register Purchase Event Data Source](#data-source)
+    + [Subscription Purchase Event Data Source](#ars-data-source)
+    + [In Apps Purchase Event Data Source](#inapps-data-source)
+  * [Register Validation Results Listeners](#validation-callbacks)
+    + [Subscription Validation Result Listener](#ars-validation-callbacks)
+    + [In Apps Validation Result Listener](#inapps-validation-callbacks)
+  * [ Testing The Integration](#testing)
+  * [Full Code Example](#example)
+<!-- TOC end -->
 
-- Android AppsFlyer SDK **6.6.0**
+## <a id="plugin-build-for"> This Module Is Built For
 
-### 📲 Adding the SDK to your project
+- Android AppsFlyer SDK **6.8.0**
+
+## <a id="install-sdk">  Adding The SDK To Your Project
 
 Add to your build.gradle file:
 
 ```
-implementation 'com.appsflyer:purchase-connector:0.1.0'
+implementation 'com.appsflyer:purchase-connector:0.2.0'
 implementation 'com.android.billingclient:billing:$play_billing_version'
 ```
 
-where play_billing_version is 3.x.x. Note, version 4 of Billing Client is not supported!
+where play_billing_version is 3.x.x. Note, versions 4 and 5 of Billing Client are not supported!
+
+## <a id="basic-integration"> Basic Integration Of The SDK
+### <a id="create-instance"> Create PurchaseClient Instance 
+Create an instance of this SDK to configure (in the following steps) for observing and validating transactions in your app. </br>
```

Comment:
not SDK, let's stick to the name Connector / Module. 
The word SDK should be used only for the AppsFlyer SDK context.